.npmrc

```
@@ -0,0 +1,2 @@
+@strg-at:registry=https://npm.pkg.github.com
+save-exact=true
```

```suggestion
save-exact=true
audit-level=high
```
is set in some repos